packages/lit-helpers/src/read-only-properties-mixin.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Enables the nofity option for properties to fire change notification events
+ */
+export function ReadOnlyPropertiesMixin(superclass) {
```

Comment:
not deduped? what happens if it is applied multiple times?

we have a limitation that it can only be at the root level.. but what happens if you apply it later again? do we wanna test what happens? or prevent it with dedupeMixin?

Reply:
We can dedupe.

Base class requirement is a bug not a feature, a later pr should fix that one

Reply:
I'm not understanding this point. Why you say this is a bug or even a limitation? I understand that you should apply the mixin in those classes or mixins in which you want to have read only properties, and that's why we need to dedupe it.

Reply:
The limitation is the the mixin has to be applied to `LitElement`, it doesn't work if applied further up the chain